packages/cli/src/cmds/preview/errors.ts

```diff
@@ -99,3 +99,11 @@ export function SPFxNotSupported(): UserError {
     "SPFxNotSupported"
   );
 }
+
+export function DependencyCheckerFailed(): UserError {
+  return returnSystemError(
+    new Error("dependency checker failed."),
+    constants.cliSource,
+    "DependencyCheckerFailed"
```

Comment:
Any help link for this error?

Reply:
It's overview error, inside will display error details and helplink.